README.md

```diff
@@ -1,3 +1,11 @@
-## Immutable Ledger used in Plenum.. 
+## Immutable Ledger used in Plenum. 
+
+This codebase provides a very simple, python-based immutable distributed
```

Comment:
Thank you for helping me be more accurate, Lovesh. I will submit a revised PR instead.